plenum/client/client.py

```diff
@@ -27,9 +27,9 @@
 from plenum.common.signer import Signer
 from plenum.common.stacked import NodeStack
 from plenum.common.startable import Status, LedgerState, Mode
-from plenum.common.txn import REPLY, POOL_LEDGER_TXNS, \
-    LEDGER_STATUS, CONSISTENCY_PROOF, CATCHUP_REP, REQACK, REQNACK
-from plenum.common.types import Reply, OP_FIELD_NAME, f, HA, \
+from plenum.common.constants import REPLY, POOL_LEDGER_TXNS, \
```

Comment:
I agree with you. While doing my changes, I just processed a TODO to rename txn.py to constants.py, because previously it was even worse: we had a module with all constants, but the name of the module was confusing (txn.py contained not only constants related to transactions).
I will create a task to re-factor usage of constants in the code (because I think it's a re-factoring task which is not in the scope of the current story). 
In addition to splitting the constants file, I believe we need more well-defined types and enumerations, not just string constants.